.github/workflows/Release-all.yml

```diff
@@ -147,7 +147,7 @@ jobs:
           if [[ '${{env.prerelease}}' == 'true' ]]
```

Comment:
the variable {{env.prerelease}} can also be removed by checking the current branch.
If current_branch=master,  it is release version.
It can be fixed in another PR.